app/models/user.rb

```diff
@@ -61,6 +61,8 @@ class User < ApplicationRecord
   has_many :user_avatars, dependent: :destroy, inverse_of: :user
 
   scope :confirmed_email, -> { where.not(confirmed_at: nil) }
+  scope :newcomers, -> { where(wca_id: nil) }
+  scope :newcomer_month_eligible, -> { where(wca_id: nil).or(where('wca_id LIKE ?', "#{Time.current.year}%")) }
```

Comment:
You can actually define this as `-> { newcomers.or(...) }`

Reply:
Ooh, that's nice - changed!